### PR TITLE
Switch from LlamaKit to antitypical/Result

### LINKED ISSUE #41: Switch from LlamaKit to antitypical/Result

### PR DESCRIPTION
**Pull request by mdiep**
Fixes #41.

### CODE REVIEW

**Comment by jspahrsummers** [comment]
dependencies

**Comment by mdiep** [comment]
Gonna close this until https://github.com/antitypical/Result/pull/29 is merged and released.

**Comment by robrix** [comment]
https://github.com/antitypical/Result/pull/29 has been merged

**Comment by mdiep** [comment]
This is ready to go now. :sparkles:

File: SwiftGit2/Repository.swift [hunk 1]
```diff
@@ -26,11 +26,11 @@ final public class Repository {
 		let result = git_repository_open(&pointer, URL.fileSystemRepresentation)
 		
 		if result != GIT_OK.value {
-			return failure(libGit2Error(result, libGit2PointOfFailure: "git_repository_open"))
+			return Result.failure(libGit2Error(result, libGit2PointOfFailure: "git_repository_open"))
 		}
 		
 		let repository = Repository(pointer)
-		return success(repository)
+		return Result.success(repository)
```
Comment (robrix) [inline SwiftGit2/Repository.swift]: You can also write `.success` here, if that would be more agreeable. (I don’t have strong feelings either way myself.)

**Comment by robrix** [comment]
Just the one comment which I’ll defer to you on. Merge away if you’d rather not address it in this PR/at all!

File: SwiftGit2/Repository.swift (hunk 1 above)
Comment (mdiep) [inline SwiftGit2/Repository.swift]: I think I'll stick with `Result.success` for now.

Comment (robrix) [inline SwiftGit2/Repository.swift]: :+1:

**Pull request merged**